### ISSUE TITLE
Low bitrate downloaded songs from Spotify

### ISSUE THREAD

**Issue by Bogdan7c** [PrajwalVandana/maestro-cli#16]
Is it possible to set up Maestro to convert to mp3 320 kbps?

**Comment by PrajwalVandana** [comment]
The Spotify songs are actually downloaded from YouTube using yt-dlp; it’s just that the Spotify metadata is used to search for the song from YouTube. I believe you MIGHT need YT premium to be able to access 320 kbps; see these Reddit answers for more: 
- https://www.reddit.com/r/youtubedl/comments/ulwtlx/i_want_to_download_audio_only_does_youtubedl/
- https://www.reddit.com/r/youtubedl/comments/1933vey/ytdlp_for_320kbps_mp3/

I can definitely add a `--audio-quality` flag to forward to yt-dlp to allow choosing bitrate in 2.0.4

I’ll also look into whether one needs yt premium for the higher quality audio

**Comment by PrajwalVandana** [comment]
Never mind, looks like you can get 320k bitrate converted mp3! Will add this feature in 2.0.4

**Comment by PrajwalVandana** [comment]
In the meantime @Bogdan7c, you can use the command
```
spotdl download <INSERT LINK/QUERY> --output "{title}.{output-ext}" --format mp3 --bitrate 320k
```
and then `maestro add -M <PATH TO DOWNLOADED SONG>`

**Comment by PrajwalVandana** [comment]
Just released 2.0.4; use `maestro add -S <LINK/QUERY> -q 320k` to download 320k mbps mp3 from Spotify (works with `-Y` for YouTube too)
